recipes/zlib/1.2.11/conanfile.py

```diff
@@ -81,14 +82,17 @@ def _build_zlib_cmake(self):
         cmake.build(build_dir=".", target=make_target)
 
     def _build_zlib(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
         with tools.chdir(self._source_subfolder):
             # https://github.com/madler/zlib/issues/268
             tools.replace_in_file('gzguts.h',
                                   '#if defined(_WIN32) || defined(__CYGWIN__)',
                                   '#if defined(_WIN32) || defined(__MINGW32__)')
-            if self.settings.os == "iOS":
-                tools.replace_in_file("gzguts.h", '#ifdef _LARGEFILE64_SOURCE',
-                                      '#include <unistd.h>\n\n#ifdef _LARGEFILE64_SOURCE')
+            # if self.settings.os == "iOS":
+            #     tools.replace_in_file("gzguts.h", '#ifdef _LARGEFILE64_SOURCE',
+            #                           '#include <unistd.h>\n\n#ifdef _LARGEFILE64_SOURCE')
```

Comment:
```suggestion
```

Reply:
this is actually an interesting one, someone added that, but I wonder how it was tested, and why it was added, when the iOS build did not work. 
So I out-commented it , to see if everything works without that, and it seems to do so, but I will double check again that tomorrow and give it a second look.

Reply:
maybe @lasote , who added those lines initially , can tell us if they are copied  from some other build / recipe and if there is maybe a reference wherefore they are needed
because building the lib without this 'patch' works fine for me

Reply:
I committed now the removal of those lines, since I have build and running software using this package for iOS on the simulator and an actual device without any problems.

@uilianries , or anyone else, since this fixes a real bad bug, building libraries in a wrong architecture without any feedback that something is wrong, could you please do a review